bb-service/src/main/java/ru/codesquad/breed/mapper/BreedMapper.java

```diff
@@ -1,25 +1,40 @@
 package ru.codesquad.breed.mapper;
 
-import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
 import ru.codesquad.breed.Breed;
 import ru.codesquad.breed.dto.BreedFullDto;
 import ru.codesquad.breed.dto.BreedNewDto;
 import ru.codesquad.breed.dto.BreedShortDto;
 import ru.codesquad.breed.enums.FurType;
 import ru.codesquad.util.enums.EnumUtil;
+import ru.codesquad.util.enums.Gender;
 import ru.codesquad.util.enums.PetType;
 
-@Mapper(componentModel = "spring",
-        imports = {PetType.class, FurType.class, EnumUtil.class},
-        disableSubMappingMethodsGeneration = true)
-public interface BreedMapper {
+public class BreedMapper {
```

Comment:
Думаю стоит использовать аннотацию @UtilityClass

Reply:
Согласно документации ломбок эта аннотация относится к экспериментальным:

> Experimental because:
Some debate as to whether it's common enough to count as boilerplate.
Due to limitations in javac, currently non-star static imports cannot be used to import stuff from @UtilityClasses; don't static import, or use star imports.
Current status: negative - Currently we feel this feature cannot move out of experimental status due to fundamental issues with non-star static imports.

[Источник](https://projectlombok.org/features/experimental/UtilityClass)

Ее преимущества не настолько велики, чтобы в ней была большая необходимость, поэтому я бы оставил как есть.

Reply:
Иногда стоит рискнуть, и довериться разработчикам )